server/routes/reviewer.ts

```diff
@@ -81,3 +67,23 @@ export = function CreateReviewRoutes({ offenderService, reportDetailBuilder, rev
     },
   }
 }
+
+function viewIncidents(page, tab, reviewService) {
+  return async (req: Request, res: Response): Promise<void> => {
+    const { prisonerName, prisonNumber, reporter, dateFrom, dateTo } = req.query
+    const query = removeKeysWithEmptyValues({
+      prisonerName,
+      prisonNumber,
+      reporter,
+      dateFrom: dateFrom ? parseDate(dateFrom, 'D MMM YYYY') : null,
+      dateTo: dateTo ? parseDate(dateTo, 'D MMM YYYY') : null,
+    })
+
+    const reports = await reviewService.getReports(res.locals.user.username, res.locals.user.activeCaseLoadId, query)
```

Comment:
Think we should be splitting this call out into 2 separate methods (getIncompleteReports and getCompletedReports) on the service - we don't want to return all incidents to the template. This will become even more apparent when trying to apply pagination

Reply:
done